de.akaflieg_freiburg.enroute.json

```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "de.akaflieg_freiburg.enroute",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "enroute",
+    "finish-args": [
+	"--filesystem=host",
```

Comment:
why does this need the full host access? not impossible to have it - just we need to understand if it's possible without.